tests/models/test_default_evaluator.py

```diff
@@ -294,6 +297,42 @@ def test_svm_classifier_evaluation(svm_model_uri, breast_cancer_dataset):
     }
 
 
+def test_pipeline_model_kernel_explainer_on_categorical_features(pipeline_model_uri):
+    data, target_col = get_pipeline_model_dataset()
+    with mlflow.start_run() as run:
+        evaluate(
+            pipeline_model_uri,
+            data[0::3],
+            model_type="classifier",
+            targets=target_col,
+            dataset_name="pipeline_model_dataset",
+            evaluators="default",
+            evaluator_config={"explainability_algorithm": "kernel"},
+        )
+    _, _, _, artifacts = get_run_data(run.info.run_id)
+    assert {
+        "shap_beeswarm_plot_on_data_pipeline_model_dataset.png",
+        "shap_feature_importance_plot_on_data_pipeline_model_dataset.png",
+        "shap_summary_plot_on_data_pipeline_model_dataset.png",
+    }.issubset(artifacts)
```

Comment:
```suggestion
    run_data = get_run_data(run.info.run_id)
    assert {
        "shap_beeswarm_plot_on_data_pipeline_model_dataset.png",
        "shap_feature_importance_plot_on_data_pipeline_model_dataset.png",
        "shap_summary_plot_on_data_pipeline_model_dataset.png",
    }.issubset(run_data.artifacts)
```

nit: this might be cleaner